src/documentation/io/authoring-java.md

```diff
@@ -10,6 +10,13 @@ permalink: /documentation/io/authoring-java/
 
 > Note: This guide is still in progress. There is an open issue to finish the guide: [BEAM-1025](https://issues.apache.org/jira/browse/BEAM-1025).
 
+## Example I/O Transforms
+Currently, the implemented IO transforms show a variety of different
+styles. These are the suggested examples to follow:
+* DatastoreIO - ParDo based database read, write, and good example of transform API
```

Comment:
I'm not totally clear what "good example of transform API" really means. my guess is something like:
"ParDo-based database read and write that conforms to the PTransform style guide"
or is it something different?